.github/workflows/ci.yml

```diff
@@ -51,714 +51,765 @@ jobs:
-
-      - name: Download artifact
+      - name: Download image artifact
         uses: actions/download-artifact@v4
         with:
           name: image
           path: /tmp/
 
-      - name: Load image
-        run: |
-          docker load --input /tmp/image.tar
-          docker image ls -a
-
-      - name: Create license
-        run: echo '${{secrets.LICENSE_KEY}}' > ./misc/license.lic
-
-      - name: Run Database
-        run: |
-          docker run --network=host -v ${PWD}/misc:/misc --env RS_API_TOKEN=${{matrix.token}}  \
-            --name reduct                                     \
-            --env RS_CERT_PATH=${{matrix.cert_path}}          \
-            --env RS_LICENSE_PATH=${{matrix.license_path}}    \
-            --env RS_CERT_KEY_PATH=/misc/privateKey.key       \
-            --env RS_CORS_ALLOW_ORIGIN="https://first-allowed-origin.com, https://second-allowed-origin.com"  \
-            -d ${{github.repository}}
-          sleep 5
-
-      - name: Build API tests
-        run: |
-          docker login -u ${{ secrets.DOCKER_USER }} -p ${{ secrets.DOCKER_TOKEN }}
-          docker build -t api ./integration_tests/api
-
-      - name: Run API tests
-        run: docker run --network=host
-          --env API_TOKEN=${{matrix.token}}
-          --env LICENSE_PATH=${{matrix.license_path}}
-          --env STORAGE_URL=${{matrix.url}} api
-
+      - name: Load Docker image
+        run: docker load --input /tmp/image.tar
 
-      - name: Print docker logs
-        if: always()
-        run: docker logs reduct
-
-  migration_test:
-    name: Migration Tests
-    runs-on: ubuntu-latest
-    needs:
-      - unit_tests
-      - build
-    strategy:
-      matrix:
-        version: [ 'latest', 'v1.14.0', 'v1.13.5', 'v1.12.4', 'v1.11.2', 'v1.10.1', 'v1.9.5', 'v1.8.2', 'v1.7.3' ]
-    env:
-      RS_API_TOKEN: XXXX
-    steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-python@v5
-        with:
-          python-version: 3.12
+      - name: Clone external repository
+        run: git clone https://github.com/reductstore/sdk-benchmarks.git
 
-      - name: Run latest stable ReductStore
+      - name: Replace files
         run: |
-          docker login -u ${{ secrets.DOCKER_USER }} -p ${{ secrets.DOCKER_TOKEN }}
-          docker run --network=host -v ./data:/data --env RS_API_TOKEN=${RS_API_TOKEN} --name latest -d reduct/store:${{matrix.version}}
-          sleep 5
-          docker logs latest
+          cp .github/files/docker-compose.yml sdk-benchmarks/
+          cp .github/files/main.py sdk-benchmarks/python/
 
-      - name: Upload data
+      - name: Run docker compose with current image
+        working-directory: ./sdk-benchmarks
         run: |
-          pip install -r ./integration_tests/data_check/requirements.txt
-          python3 ./integration_tests/data_check/uploader.py
+          REDUCT_IMAGE=${{github.repository}}:latest docker compose up --exit-code-from bench-py
```

Comment:
don't use docker-compose, let's do like we did in the  `replicaion_test` or `migration_test jobs`